test/e2e/node/stress_probe.go

```diff
@@ -0,0 +1,200 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package node
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	admissionapi "k8s.io/pod-security-admission/api"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+)
+
+const (
+	probeTestInitialDelaySeconds = 15
+	numTestPods                  = 1
+	numContainers                = 400
+	containerPort                = 2000
+	defaultObservationTimeout    = time.Minute * 4
+)
+
+type containerConfig struct {
+	Name         string
+	ProbeType    string
+	ContainerIdx int
+}
+
+var _ = SIGDescribe("[Serial] [Slow] [Disruptive] Stress test probes", func() {
```

Comment:
@rphillips we asked to add all these tags to indicate these tests are not meant to run by default, you can see that it has `Disruptive` already